### ISSUE TITLE
Allow the customization of colors for line symbols

### ISSUE THREAD

**Issue by palewire** [datadesk/datawrapper-migration#78]
As [this example](https://app.datawrapper.de/chart/Ty7MH/visualize?mode=print) shows, symbols are fixed to the same color as the line.

![Screenshot from 2020-01-19 17-42-10](https://user-images.githubusercontent.com/9993/72692888-35092380-3ae3-11ea-9ed4-019744a29b11.png)

We would like the ability to customize the color of these symbols. Our print style is for the final symbol to be black, so in this case we had to download the PDF from datawrapper and make the following customization in Adobe Illustrator. We'd prefer to handle it in your application. 

![Screenshot from 2020-01-19 17-42-55](https://user-images.githubusercontent.com/9993/72692940-77326500-3ae3-11ea-9352-4efe76899ea9.png)

**Comment by elanals** [comment]
I'll put the request forward for discussion and let you know the results. 

If you like I can set line symbols to always be black for your print theme. Alternatively you can include another column in your dataset with just the points that should have symbols, set the line color to black, and the line width to 0px (if there are more than one points, otherwise it doesn't matter)

**Comment by elanals** [comment]
![image](https://user-images.githubusercontent.com/19191012/72819483-a248c000-3c6d-11ea-8092-90b2d3c43ec2.png)

**Comment by elanals** [comment]
![image](https://user-images.githubusercontent.com/19191012/72819537-bc829e00-3c6d-11ea-8678-13ad6dc4fdd5.png)

**Comment by palewire** [comment]
Yes, can you please set it to be always black for our print theme until we get a deeper fix.
